src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/HighRiskBreachLayout.tsx

```diff
@@ -0,0 +1,256 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import styles from "./HighRiskBreachLayout.module.scss";
+import CreditCardIllustration from "../images/high-risk-data-breach-credit-card.svg";
+import BankAccountIllustration from "../images/high-risk-data-breach-bank-account.svg";
+import pinIllustration from "../images/high-risk-data-breach-pin.svg";
+import SocialSecurityNumberIllustration from "../images/high-risk-data-breach-ssn.svg";
+import NoBreachesIllustration from "../images/high-risk-breaches-none.svg";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { useL10n } from "../../../../../../../hooks/l10n";
+import { ResolutionContentLayout } from "../ResolutionContentLayout";
+
+type HighRiskBreachLayoutProps = {
+  typeOfBreach: "creditCard" | "ssnBreaches" | "bankAccount" | "pin" | "none";
+  breachData: GuidedExperienceBreaches;
+};
+
+export const HighRiskBreachLayout = (props: HighRiskBreachLayoutProps) => {
+  const l10n = useL10n();
+  const highRiskDataBreaches = props.breachData.highRisk;
+  let exposedData: SubscriberBreach[] = [];
+
+  if (props.breachData) {
+    switch (props.typeOfBreach) {
+      case "ssnBreaches":
+        exposedData = highRiskDataBreaches.ssnBreaches;
+        break;
+      case "creditCard":
+        exposedData = highRiskDataBreaches.creditCardBreaches;
+        break;
+      case "bankAccount":
+        exposedData = highRiskDataBreaches.bankBreaches;
+        break;
+      case "pin":
+        exposedData = highRiskDataBreaches.pinBreaches;
+        break;
+      default:
+        "none";
+        break;
+    }
+  }
+
+  // TODO: Make locale location-sensitive in the future
+  const dateFormatter = new Intl.DateTimeFormat("en-US", {
```

Comment:
To clarify: you'd still use `Intl.DateTimeFormat`, but the first argument would be decided by `getLocale`.

The only reservation I'd have here is that for these specific strings we might not actually have translations into the given locale (since we're not submitting them to Pontoon), which might result in e.g. Spanish date formats in an English UI.  But then again hardcoding `en-US` now might result in the reverse problem if we do ever localise, so :shrug:

Reply:
Also noting that whatever you decide to do here, make sure the `ListFormatter` below does the same.